docs/getting_started/installation.rst

```diff
@@ -27,7 +27,6 @@ storage bundles. The official storage bundles are:
 
 * `SonataDoctrineORMAdminBundle`_ (integrates the Doctrine ORM);
 * `SonataDoctrineMongoDBAdminBundle`_ (integrates the Doctrine MongoDB ODM);
-* `SonataPropelAdminBundle`_ (integrates Propel);
 * `SonataDoctrinePhpcrAdminBundle`_ (integrates the Doctrine PHPCR ODM).
 
 You can download them in the same way as the SonataAdminBundle. For instance,
```

Comment:
Here is where it picks sonataDoctrineORMAdmin to give the example

Reply:
I understand that, but that can also be found on `SonataDoctrineORMAdminBundle` installation page if you choose to use that, do we also need it here? that is why I said that `tip` part was enough, but if you think that it should also be here, I am ok with that.